com.unity.mobile.notifications/Runtime/Android/AndroidNotification.cs

```diff
@@ -177,6 +177,11 @@ public DateTime CustomTimestamp
             }
         }
 
+        /// <summary>
+        /// Set this notification to be shown when app is in the foreground.
+        /// </summary>
+        public bool ShowInForeground { get; set; }
```

Comment:
Unfortunately this is a breaking change.
Mu suggestion is to back the ShowInForeground property by m_HideInForeground boolean, so that if you don't set the property, the default behavior is as it used to be.